src/components/migrated-legacy.js

```diff
@@ -8,11 +8,13 @@ import * as _ from 'lodash';
 import ErrorMessage from './error-message';
 
 import tooltips from '../constants/tooltips';
+import coinConfig from 'constants/coin-config';
 
-const formTooltips = tooltips.migratedBch;
+const formTooltips = tooltips.migratedLegacy;
 
-class MigratedBchRecoveryForm extends Component {
+class MigratedRecoveryForm extends Component {
   state = {
+    coin: 'bch',
```

Comment:
So what's up... by default, this class is configured to migrate BCH, but can be overridden if specified by the user?

Reply:
yes exactly

Reply:
so BCH is the default for the drop-down menu